lib/mongo_client.js

```diff
@@ -241,28 +243,35 @@ MongoClient.prototype.close = function(force, callback) {
     force = false;
   }
 
-  const client = this;
```

Comment:
We were using arrow functions everywhere here, so I just updated everything to use `this`

Reply:
iirc the reason `client` was used is because this method has significant enough depth that `this` can't be bound to the deepest nested functions. Can you confirm, for instance, that assignment to `autoEncrypter` below is actually referencing the correct `this`?

Reply:
Confirmed that if I add `client` back in, `client === this` is true on the line before the assignment to `autoEncrypter`.